sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/QueueDescription.cs

```diff
@@ -273,11 +275,27 @@ public string UserMetadata
             }
         }
 
+        internal bool IsAnonymousAccessible { get; set; } = false;
+
+        internal bool SupportOrdering
+        { 
+            get
+            {
+                return this.internalSupportOrdering ?? !this.EnablePartitioning;
```

Comment:
>this.EnablePartitioning [](start = 56, length = 23)

I'm not sure what the relationship is between SupportOrdering and EnablePartitioning?

Reply:
If not partitioned, queues always support ordering. Topics optionally support ordering.
If partitioned, support ordering has no use. but it is part of the description.